### ISSUE TITLE
ObjectiveTrackerFrame LUA error with RC390

### ISSUE THREAD

**Issue by pauldcox** [GoldpawsStuff/AzeriteUI#16]
This is similar to issue #14 but a little different. Unlike that one, the UI loads now, but I still get this error. And the objective tracker appears to work even with it. This is on Classic with RC390 on a fresh install of the UI.

```
Message: ...s\AzeriteUI\front-end\blizzard\objectivestracker.lua:496: attempt to index global 'ObjectiveTrackerFrame' (a nil value)
Time: Fri Aug 14 04:13:06 2020
Count: 1
Stack: ...s\AzeriteUI\front-end\blizzard\objectivestracker.lua:496: attempt to index global 'ObjectiveTrackerFrame' (a nil value)
...s\AzeriteUI\front-end\blizzard\objectivestracker.lua:496: in function `PositionRetailTracker'
...s\AzeriteUI\front-end\blizzard\objectivestracker.lua:573: in function `?'
...face\AddOns\AzeriteUI\back-end\libraries\message.lua:63: in function `Fire'
...erface\AddOns\AzeriteUI\back-end\libraries\event.lua:71: in function <...erface\AddOns\AzeriteUI\back-end\libraries\event.lua:68>

Locals: self = <table> {
 OnInit = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:581
 IsClassicTBC = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\clientbuild.lua:64
 IsRetail = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\clientbuild.lua:69
 IsEventRegistered = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\message.lua:200
 IsRetailShadowlands = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\clientbuild.lua:79
 CreateDriver = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:510
 modules = <table> {
 }
 GetModule = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\module.lua:669
 IsClassic = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\clientbuild.lua:59
 CreateFrame = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\frame.lua:351
 StyleClassicLog = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:195
 SetAspectRatio = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\frame.lua:391
 StyleRetailTracker = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:411
 CreateTextureCache = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\frame.lua:382
 GetFrame = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\frame.lua:375
 IsEnabled = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\module.lua:717
 Fire = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\message.lua:52
 GetCurrentClientBuild = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\clientbuild.lua:90
 GetClientBuildByPatch = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\clientbuild.lua:85
 InitRetailTracker = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:435
 IsInitialized = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\module.lua:713
 RegisterUnitEvent = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\message.lua:104
 libraries = <table> {
 }
 NewModule = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\module.lua:612
 layout = <table> {
 }
 RegisterEvent = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\message.lua:73
 moduleLoadPriority = <table> {
 }
 ForAll = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\module.lua:678
 OnEvent = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:564
 PositionRetailTracker = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:492
 OnEnable = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:597
 UnregisterAllEvents = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\message.lua:177
 IsRetailBFA = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\clientbuild.lua:74
 StyleClassicTracker = <function> defined @Interface\AddOns\AzeriteUI\front-end\blizzard\objectivestracker.lua:210
 frame = <unnamed> {
 }
 UnregisterEvent = <function> defined @Interface\AddOns\AzeriteUI\back-end\libraries\me
```

**Comment by goldpaw** [comment]
Different error but same problem, yes. It's a post update meant to fix the position of the tracker after the saved settings with blizzard's position of it has been loaded. And surprise surprise... even though I do check if the tracker is loaded in the rest of the that module, this one somehow skipped my not so keen attention! 

I'll get it fixed asap! :)

**Comment by goldpaw** [comment]
Fixed in 2.0.393-RC.
